app/src/interfaces/_system/system-filter/input-component.vue

```diff
@@ -152,7 +152,7 @@ export default defineComponent({
 		return { displayValue, width, t, emitValue, inputEl, inputPattern, dateTimeMenu };
 
 		function emitValue(val: unknown) {
-			const mustache = new RegExp(/({{.*?}})/g);
+			const mustache = new RegExp(/^({{.*?}})$/);
```

Comment:
@rijkvanzanten Should "multi tags" be allowable in the system filter context for insights? Eg: users can form custom strings `{{var1}}_{{var2}}`.

I wonder if a property should be added to `allowVariables` on `system-filter.vue` as it is currently only supported in insights. 🤔

The regex could be updated to along the lines of `({{[\w\d_]+?}})` as only alphanumeric characters and underscores are allowed.

Reply:
The regex should definitely be updated at the very least

Reply:
Seeing we're now using this interface inside insights options as well, it should support the multi-var style too yeah